src/App.css

```diff
@@ -0,0 +1,91 @@
+body {
+  margin: 0;
+  background-color: rgba(0, 0, 0, 0.05);
+  font-family: Montserrat, sans-serif;
+}
```

Comment:
- [ ] Your buttons still do not have the correct `border`. you should remove the border on the buttons entirely.